### PR TITLE
Generate both Three Axis and Acceleration events when included in same msg

### PR DESCRIPTION
**Pull request by tpmanley**
The x,y,z attributes are often sent in a separate Attribute Report from the
accelerometer attribute. Sometimes, however, they are all sent in the same
Attribute Report. When that happens, only the accelerometer attribute was
being handled and the x,y,z attributes were not. Now they are all handled
if they arrive in the same message.

Resolves:
https://smartthings.atlassian.net/browse/DVCSMP-1271

### CODE REVIEW

**Comment by tpmanley** [comment]
Please review @workingmonk

File: devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy [hunk 1]
```diff
@@ -115,31 +115,30 @@
...
- 	else if (description?.startsWith('read attr -')) {
- 		map = parseReportAttributeMessage(description)
- 	}
+def parse(String description) {
+	Map map = [:]
+	if (description?.startsWith('catchall:')) {
+		map = parseCatchAllMessage(description)
+	}
     else if (description?.startsWith('temperature: ')) {
- 		map = parseCustomMessage(description)
```
Comment (workingmonk) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy:128]: is all of this space changes?

Comment (tpmanley) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy:128]: I tried to avoid unnecessary whitespace changes, but a few crept in. Add '?w=1' to the url of the PR to ignore the whitespace changes. I have a separate branch that fixes all the inconsistent spacing in this DTH.

**Comment by workingmonk** [comment]
is there anything remaining on this PR before we merge this in?

**Comment by tpmanley** [comment]
Just your :+1: :)

**Comment by workingmonk** [comment]
:+1: 

Can you assign the ticket to QA? and add the ticket number here. That way we can push this to prod asap.

**Comment by tpmanley** [comment]
The ticket is in the commit message and I'll assign it to QA
